libraries/Microsoft.Bot.Connector/Authentication/BotFrameworkAuthentication.cs

```diff
@@ -59,6 +59,26 @@ public virtual BotFrameworkClient CreateBotFrameworkClient()
             throw new NotImplementedException();
         }
 
+        /// <summary>
+        /// Gets the originating audience from Bot OAuth scope.
+        /// </summary>
+        /// <returns>The originating audience.</returns>
+        public virtual string GetOriginatingAudience()
```

Comment:
We should try to get rid of this in a separate PR

Reply:
We will revisit this in R14.

Reply:
We should try to remove it earlier if time allows so we don't have to deprecate an existing public method